packages/helm-charts/celo-fullnode/values.yaml

```diff
@@ -82,3 +82,11 @@ genesis:
   network: baklavastaging
   useGenesisFileBase64: false
   genesisFileBase64: ""
+
+# If Prometheus is true, metrics and pprof set to true
+prometheus: true
+metrics: true
```

Comment:
Right now `metrics` isn't used, what do you think of changing the template `common.full-node-container` to conditionally add the `--metrics` flag to the additional flags? We can have it default to true to preserve backward compatibility